build/NuSpecs/UmbracoCms.Examine.Lucene.nuspec

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata minClientVersion="4.1.0">
+        <id>Umbraco.Cms.Examine.Lucene</id>
+        <version>9.0.0</version>
+        <title>Umbraco CMS Examine Binaries</title>
+        <authors>Umbraco HQ</authors>
+        <owners>Umbraco HQ</owners>
+        <license type="expression">MIT</license>
+        <projectUrl>https://umbraco.com/</projectUrl>
+        <iconUrl>https://umbraco.com/dist/nuget/logo-small.png</iconUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>Contains the Examine assemblies needed to run Umbraco Cms. This package only contains assemblies and can be used for package development. Use the UmbracoCms package to setup Umbraco in Visual Studio as an ASP.NET Core project.</description>
+        <summary>Contains dll files required to run Examine.</summary>
+        <language>en-US</language>
+        <tags>umbraco</tags>
+        <repository type="git" url="https://github.com/umbraco/umbraco-cms" />
+        <dependencies>
+
+            <group targetFramework="netstandard2.0">
+                <dependency id="Umbraco.Cms.Core" version="[$version$]" />
+                <dependency id="Umbraco.Cms.Infrastructure" version="[$version$]" />
+                <dependency id="Newtonsoft.Json" version="[12.0.3,12.999999)" />
+                <dependency id="NPoco" version="[4.0.2,4.999999)" />
```

Comment:
Are we not missing `System.Configuration.ConfigurationManager` here? 

We have the following TODO in netcore csproj, that could be relavant

```
<!-- TODO: remove the reference to System.Configuration.ConfigurationManager when Examine/lucene dont need it-->
<PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />

```

Reply:
Hmm that's a good question, I didn't see that TODO in the netcore csproj. I just used the dependencies from the Examine.Lucene.csproj.

I don't know if we actually need to add it to the Examine.Lucene.nuspec file though, since Umbraco.Core.csproj already has `<PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />` so it will still be installed as an implicitly referenced version from there.

On the other hand I don't think it would hurt either, if we want to be 100% sure that it's there regardless of Umbraco.Core

Reply:
Cool I guess that is another problem.. Core should not have that dependency. Seems to be because we just use an exception.

I can fix that part up